frontend/src/views/Raid.vue

```diff
@@ -256,26 +273,37 @@ export default {
         return;
       }
 
+      const canJoin = await this.canJoinRaid(this.currentCharacterId, this.selectedWeaponId);
+      if(!canJoin) {
+        this.$dialog.notify.error(`Can't join... Make sure a raid is active, your stamina/durability are both > 0
+          and selected character/weapon are not locked in the raid already.`);
+        return;
+      }
+
       try {
         console.log('Trying to join raid...');
         await this.joinRaid({ characterId: this.currentCharacterId, weaponId: this.selectedWeaponId});
+        this.selectedWeaponId = null;
         console.log('Made it to the other side at least...');
```

Comment:
console.log

Reply:
Wasn't added by me, but I'll delete all other raid realted console logs then.

Reply:
I'm not sure if they're needed for debugging or what, but unless they're needed I'd like to keep them away. thanks!